scripts/migrate.js

```diff
@@ -0,0 +1,45 @@
+const { migrate } = require('postgres-migrations');
+const path = require('path');
+
+const { NODE_ENV } = process.env;
+
+// This code is repeated a few times in our scripts. Can you factor it out into a shared helper function?
```

Comment:
This is copied in again. This really shouldnt be here as any potential employer looking at this will tell in an instant that you didnt write this comment yourself